packages/jest-each/src/bind.js

```diff
@@ -23,6 +24,13 @@ const RECEIVED_COLOR = chalk.red;
 const SUPPORTED_PLACEHOLDERS = /%[sdifjoOp%]/g;
 const PRETTY_PLACEHOLDER = '%p';
 const INDEX_PLACEHOLDER = '%#';
+const PRIMITIVES = new Set([
```

Comment:
Not blocking at all, but we might consider moving this to `jest-get-types` (e.g.: `isPrimitiveType(getType(value))`).

Reply:
@rubennorte yup good shout, I'll send another PR for this change so we can merge this PR in the meantime :)